src/components/Table/Table.jsx

```diff
@@ -0,0 +1,350 @@
+				'&-hasIcon': hasIcon,
+				'&-hasButton': hasButton,
+				'&-isSortable': (isSortable === false ? isSortable : (isSorted || isSortable)),
+				'&-isSorted': isSorted,
+			}, className)}>
+				{isSorted ? (
+					<ul className={boundClassNames('&-isSorted-container')}>
+						<li className={boundClassNames('&-isSorted-title')}>{children}</li>
+						<li className={boundClassNames('&-isSorted-caret')}><CaretIcon className={boundClassNames('&-sort-icon')} direction={sortDirection} size={6}/></li>
+					</ul>
+				) : children}
+			</th>
+		);
+	}
+}));
+
+/**
+ * `Td` renders <td>.
+ */
+const Td = React.createClass(createLucidComponentDefinition({
+	propTypes: {
+		/**
+		 * Aligns the content of a cell. Can be `left`, `center`, or `right`.
+		 */
+		align: string,
+		/**
+		 * Should be `true` when the cell has a checkbox.
+		 */
+		hasCheckbox: bool,
+		/**
+		 * Should be `true` when the cell has an icon.
+		 */
+		hasIcon: bool,
+		/**
+		 * Should be `true` when the cell has a button.
+		 */
+		hasButton: bool,
+		/**
+		 * Should be `true` to render a right border.
+		 */
+		hasBorderRight: bool,
+		/**
+		 * Should be `true` to render a left border.
+		 */
+		hasBorderLeft: bool,
+		/**
+		 * Should be set to `true` on the second cell in a table where the first cell has a rowspan greater than 1.
+		 */
+		isAfterRowSpan: bool
+	},
+	getDefaultProps() {
+		return {
+			align: 'left',
+			hasCheckbox: false,
+			hasIcon: false,
+			hasButton: false,
+			hasBorderRight: false,
+			hasBorderLeft: false,
+		};
+	},
+	render() {
+		const {
+			className,
+			align,
+			hasCheckbox,
+			hasIcon,
+			hasButton,
+			rowSpan,
+			hasBorderRight,
+			hasBorderLeft,
+			isAfterRowSpan
+		} = this.props;
+
+		return (
+			<td {...this.props} className={boundClassNames(
+				'&-cell', {
+				'&-align-left': align === 'left',
+				'&-align-center': align === 'center',
+				'&-align-right': align === 'right',
+				'&-hasCheckbox': hasCheckbox,
+				'&-hasIcon': hasIcon,
+				'&-hasButton': hasButton,
+				'&-hasRowSpan': !_.isNil(rowSpan),
+				'&-hasBorderRight': hasBorderRight,
+				'&-hasBorderLeft': hasBorderLeft,
+				'&-isAfterRowSpan': isAfterRowSpan
+			}, className)} />
+		);
+	}
+}));
+
+/**
+ *
+ * {"categories": ["table"]}
+ *
+ * `Table` provides the most basic components to create a lucid table.
+ * It is recommended to create a wrapper around this component rather than using it directly in an app.
+ */
+const Table = React.createClass(createLucidComponentDefinition({
+	displayName: 'Table',
+
+	statics: {
+		Thead,
+		Tbody,
+		Tr,
+		Th,
+		Td,
+	},
+
+	propTypes: {
+		/**
+		 * Styles that are passed through to the root container.
+		 */
+		style: object,
+
+		/**
+		 * Class names that are appended to the defaults.
+		 */
+		className: string,
+
+		/**
+		 * Adjusts the style of the table to have more spacing within the table cells
+		 */
+		isComfortable: bool,
+	},
+
+	getDefaultProps() {
+		return {
+			isComfortable: false,
```

Comment:
would it make sense to use a more descriptive key name like `hasExtraWhitespace` ?